### ISSUE TITLE
[bug] screen reader does not read all newly rendered text

### ISSUE THREAD

**Issue by kennyzhen13** [microsoft/accessibility-insights-web#4211]
## Describe the bug
This bug report describes the issue described under **Context** in the PR description of #4170. 

While using MacOSX VoiceOver, if new text is rendered in more than one area, only one of those texts is read aloud by the screen reader. In this specific instance, a button click renders two new texts: (1) a block of text under the button and (2) a toast message pop-up.  

**To Reproduce**
1. Fetch from the branch in #4170.
2. Build and run AI-Web locally.
3. Turn on screen reader software. I used MacOSX VoiceOver.
4. Open [this webpage](http://www.washington.edu/accesscomputing/AU/before.html). 
5. Click on the Accessibility Insights for Web Dev plug-in.
6. Turn on the **Automated Checks** setting.
7. Click on any error icon.
8. In the dialog window, click on **Copy failure details** and take note of the screen reader logs.

## Expected behavior

Currently unsure about the expected behavior.

## Context (please complete the following information)

- OS Name & Version: macOS Catalina 10.15.7
- AI-Web Version & Environment: Dev version
- Browser Version: Google Chrome, version 90.0.4430.93 (Official Build) (x86_64)
- Target Page: http://www.washington.edu/accesscomputing/AU/before.html

## Are you willing to submit a PR?
Yes

## Did you search for similar existing issues?
No

## Additional context
The target page must be from an insecure origin (**http://** instead of **https://**)

**Comment by kennyzhen13** [comment]
@ferBonnin OK I can fix this since it was a part of my PR #4170

**Comment by ferBonnin** [comment]
awesome! thanks @kennyzhen13

**Comment by jalkire** [comment]
Hey @kennyzhen13, thanks for fixing this! Feel free to close out the issue if you feel that it is resolved 🙂
